courses/models_test.py

```diff
@@ -25,8 +25,10 @@ def test_towebhook(self):
         test to_webhook implementation returns valid json object
         """
         course = CourseFactory.build()
-        json.dumps(course.to_webhook())
-        ex_pk = course.to_webhook()['external_pk']
+        out = course.to_webhook()
+        json.dumps(out)
```

Comment:
If it's here to verify that JSON serialization doesn't cause an exception, can you add a comment saying that?